server/src/test/java/org/elasticsearch/index/mapper/BooleanFieldTypeTests.java

```diff
@@ -24,6 +24,9 @@
 import org.junit.Before;
 
 public class BooleanFieldTypeTests extends FieldTypeTestCase {
+
+    protected static final String BOOLEANFIELD = "booleanField1";
```

Comment:
the BFTT uses this so its clear the field is pointing to a boolean field. This is particular useful for AliasFieldTypeBooleanFieldTypeTests so we can clearly see in the mapping/error messages whether we are seeing the name of the boolean or the alias.

THis same "refactoring" has been applied to all XXXFieldTypeTests and the matching sub class that uses aliases.